gulpfile.js

```diff
@@ -11,14 +11,12 @@ const del = require('del');
 const NwBuilder = require('nw-builder');
 const innoSetup = require('@quanle94/innosetup');
 const deb = require('gulp-debian');
-const conffiles = './test/configs';
```

Comment:
This was added in #3155 - need to check if lunix debian file is still being created.

Reply:
This literally does nothing. Do you mean https://github.com/betaflight/betaflight-configurator/blob/4890ca2892d3b6642c46cb72fc5991b776d08c69/gulpfile.js#L752 is needed?

Reply:
Yes according the example in the updated library